pandas/io/html.py

```diff
@@ -1027,6 +1030,7 @@ def read_html(
     extract_links: Literal[None, "header", "footer", "body", "all"] = None,
     dtype_backend: DtypeBackend | lib.NoDefault = lib.no_default,
     storage_options: StorageOptions = None,
+    skip_blank_lines: bool = True,
```

Comment:
How does this relate to fixing the bug? Generally to introduce new parameters to a function you'd have to open an enhancement issue.

Reply:
This argument is needed because this function might need to specify the option whether they want to consider columns with just spaces as a valid column or not. If this is true, any column which is blank would be skipped as before. [Here](https://github.com/pandas-dev/pandas/blob/69fe98dda091cae71ea699be3f44926406851a2c/pandas/io/parsers/python_parser.py#L744)is the place where this is checked to remove whitespaced lines